pkg/storaged/crypto-keyslots.jsx

```diff
@@ -56,7 +56,7 @@ function get_tang_adv(url) {
     return cockpit.spawn(["curl", "-sSf", url + "/adv"], { err: "message" })
             .then(JSON.parse)
             .catch(error => {
-                return cockpit.reject(error.toString().replace(/^curl: \([0-9]+\) /, ""));
+                return Promise.reject(error.toString().replace(/^curl: \([0-9]+\) /, ""));
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18487-20230310-143650-f841503b-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)